### DISCUSSION TITLE
Guys how to create a user and login by email

### DISCUSSION THREAD

**Discussion by seoeaa** [serverpod/serverpod#662] [Q&A] [answered]
There are no documents, the module is good, but I still don’t understand how everything works, maybe someone has the code

**Comment by Splokhande** [comment, marked as answer]
Hi @seoeaa 
Refer this video
https://www.youtube.com/watch?v=8dDucblphek&list=PL0CMCjqejDDQFDNDCNJHGDnmwSbFpUWxq&index=1
